convert.sh

```
@@ -275,10 +275,14 @@ fi
 # enable flash and drivers
 FEATURES='FEATURES       += VCP '
 if [[ $(grep 'USE_SDCARD' $config) ]]; then
-    FEATURES+='SDCARD'
+    if [[ $(grep -E 'USE_SDCARD_SPI|SDCARD_SPI_INSTANCE|SDCARD_SPI_CS_PIN' $config) ]]; then
```

Shellcheck (0.11.0)

[style] Use grep -q instead of comparing output with [ -n .. ].

(SC2143)